core/src/fees.ts

```diff
@@ -23,16 +26,42 @@ export const getApproximateTransactionFees = async function (network: string): P
     };
 };
 
-export const enrichAuctionWithTransactionFees = function (
+export const enrichAuctionWithTransactionFees = async function (
     auction: Auction,
-    fees: TransactionFees
-): AuctionTransaction {
+    fees: TransactionFees,
+    network: string
+): Promise<AuctionTransaction> {
+    let totalFeeETH = new BigNumber(0);
+    let totalFeeDAI = new BigNumber(0);
+
+    try {
+        const signer = await getSigner(network);
+        const address = await signer.getAddress();
+        const isWalletAuthed = await getWalletAuthorizationStatus(network, address);
+        const isCollateralAuthed = await getCollateralAuthorizationStatus(network, auction.collateralType, address);
```

Comment:
As we can't cache those, we shouldn't call it for each auction, but rather provide it from a store?

Reply:
Can I just refer to the frontend store from the core folder? Or should I store them in a variable?

Reply:
So I made the following system:

By default it always clears the cache when retrieving the auth status. However the keeper will only clear the cache on the first request and after any recall of the participate function after any auth step was changed.